src/client/user/widgets/ConfigOption.tsx

```diff
@@ -32,55 +44,60 @@ const useStyles = makeStyles((theme) =>
     },
     trailingContainer: {
       marginTop: (props: StylesProps) =>
-        props.trailingPosition == TrailingPosition.end ? 'auto' : 'unset',
+        props.trailingPosition === TrailingPosition.end ? 'auto' : 'unset',
       marginBottom: (props: StylesProps) =>
-        props.trailingPosition == TrailingPosition.start ? 'auto' : 'unset',
+        props.trailingPosition === TrailingPosition.start ? 'auto' : 'unset',
       width: (props: StylesProps) => (props.wrapTrailing ? '100%' : 'unset'),
     },
+    regularText: {
+      fontWeight: 400,
+    },
   }),
 )
 
-export enum TrailingPosition {
-  start,
-  center,
-  end,
-  none,
-}
-
 type ConfigOptionProps = {
   title: string | React.ReactNode
+  isTitleMobileResponsive?: boolean
```

Comment:
nitpick on a wordy prop. how about simply `mobile`? details such as the title shouldn't be exposed anyway?